ping.js

```diff
@@ -0,0 +1,10 @@
+/* global CloudWatch:false - from dispatch:cloudwatch-metrics */
+
+var DEFAULT_PING_INTERVAL = 60000; // 1m
+
+var settings = Meteor && Meteor.settings && Meteor.settings.public || {};
```

Comment:
`Meteor.settings.public` should always be there, so you do not need all this protection. But this should prob just be on `Meteor.settings` since it is only on the server.